frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -55,6 +56,7 @@ const plugin: Plugin<ConsumedExtensions> = [
       id: 'persistent-storage',
       title: 'OCS PV',
       required: CEPH_FLAG,
+      blocker: FLAGS.INDEPENDENT,
```

Comment:
it would be better to call this `disallowed` to align with naming from https://github.com/openshift/console/pull/3383 or even better to rebase on top of 3383 which is close to being merged and already contains the functionality you need